CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.2] - 2021-06-08
+### Security
+- Upgrade various dependencies.
```

Comment:
It's a bit of a guess why each dependency was updated in some cases: https://github.com/NEWROPE/scnnr-js/compare/v1.0.1...master -> difficult to correctly state the details without mistakes -> I omitted the details 😓

Reply:
Ah, yeah that looks difficult. 😅 
How about separating `dependencies` and `devDependencies`?
Users only are affected by `dependencies`, and `devDependencies` affects only contributors.
Since `dependencies` only contains `axios`, we can describe details only about `axios`. And about `devDependencies`, `upgrade various dependencies for development` would be enough.

Reply:
Wow, `axios` is our only non-dev dependency! Good idea. I'll add the change. Thank you!